dftimewolf/lib/collectors/gcloud.py

```diff
@@ -137,43 +154,48 @@ def SetUp(self,
           image_project=image_project,
           image_family=image_family)
 
-      if disk_names:
-        for name in disk_names:
+    except AccessTokenRefreshError as err:
+      self.state.AddError('Something is wrong with your gcloud access token.')
+      self.state.AddError(err, critical=True)
+
+    except ApplicationDefaultCredentialsError as err:
+      self.state.AddError(
+          'Something is wrong with your Application Default Credentials. '
+          'Try running:\n  $ gcloud auth application-default login')
+      self.state.AddError(err, critical=True)
+
+  def FindDisksToCopy(self):
```

Comment:
Could this be private (_FindDisks...) and return a list of disks? It'd be a bit easier to follow the code, test etc.